activiti-cloud-acceptance-tests/activiti-cloud-acceptance-tests-core/src/main/java/org/activiti/cloud/acc/core/steps/runtime/TaskRuntimeBundleSteps.java

```diff
@@ -45,178 +44,168 @@
+        taskApiClient
+            .completeTask(id, completeTaskPayload);
     }
 
     @Step
     public void saveTask(String id, SaveTaskPayload saveTaskPayload) {
-        taskRuntimeService
-                .saveTask(id,saveTaskPayload);
+        taskApiClient
+            .saveTask(id, saveTaskPayload);
     }
+
     @Step
     public void cannotCompleteTask(String id, CompleteTaskPayload createTaskPayload) {
         assertThatRestNotFoundErrorIsThrownBy(
-                () -> taskRuntimeService.completeTask(id, createTaskPayload)
+            () -> taskApiClient.completeTask(id, createTaskPayload)
         ).withMessageContaining("Unable to find task for the given id: " + id);
     }
 
     @Step
     public CloudTask createNewTask() {
 
         CreateTaskPayload createTask = TaskPayloadBuilder
-                .create()
-                .withName("new-task")
-                .withDescription("task-description")
-                .withAssignee("testuser")
-                .build();
+            .create()
+            .withName("new-task")
+            .withDescription("task-description")
+            .withAssignee("testuser")
+            .build();
         return dirtyContextHandler.dirty(
-                taskRuntimeService.createTask(createTask));
+            taskApiClient.createNewTask(createTask).getContent());
     }
 
     @Step
     public CloudTask createNewUnassignedTask() {
 
         CreateTaskPayload createTask = TaskPayloadBuilder
-                .create()
-                .withName("unassigned-task")
-                .withDescription("unassigned-task-description")
-                .build();
+            .create()
+            .withName("unassigned-task")
+            .withDescription("unassigned-task-description")
+            .build();
         return dirtyContextHandler.dirty(
-                taskRuntimeService.createTask(createTask));
+            taskApiClient.createNewTask(createTask).getContent());
     }
 
     public CloudTask createSubtask(String parentTaskId) {
         CreateTaskPayload subTask = TaskPayloadBuilder
-                .create()
-                .withName("subtask")
-                .withDescription("subtask-description")
-                .withAssignee("testuser")
-                .withParentTaskId(parentTaskId)
-                .build();
-        return taskRuntimeService.createTask(subTask);
+            .create()
+            .withName("subtask")
+            .withDescription("subtask-description")
+            .withAssignee("testuser")
+            .withParentTaskId(parentTaskId)
+            .build();
+        return taskApiClient.createNewTask(subTask).getContent();
     }
 
-    public CollectionModel<CloudTask> getSubtasks(String parentTaskId) {
-        return taskRuntimeService.getSubtasks(parentTaskId);
+    public Collection<CloudTask> getSubtasks(String parentTaskId) {
+        return taskApiClient
+            .getSubtasks(PageRequest.of(0, 100), parentTaskId)
```

Comment:
I think `PageRequest.of(0, 100)` can be extracted inside a constant `DEFAULT_PAGEABLE` for re-use purpose.